src/main/java/io/github/jasmin/controller/UserController.java

```diff
@@ -0,0 +1,39 @@
+package io.github.jasmin.controller;
+
+import io.github.jasmin.model.User;
+import io.github.jasmin.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+    @Autowired
+    private UserService userService;
+
+    @GetMapping("")
+    public List<User> getAllUsers(){
+        return userService.getAllUsers();
+    }
+
+    @GetMapping("/{userid}")
+    public User getUserByUserid(@PathVariable String userid){
+        return userService.getUserByUserId(userid);
+    }
+
+    @PostMapping("")
```

Comment:
```suggestion
    @PostMapping
```
로 빈문자열을 빼는것도 나쁘지 않아보여요

Reply:
빈 문자열 빼도 되는군요 감사합니다!